tests/fixtures/nx-integrated/.gitignore

```diff
@@ -39,9 +39,12 @@ testem.log
 Thumbs.db
 
 .nx/cache
+.nx/workspace-data
```

Comment:
NX dependencies in nx fixture had to be updated to work with next@16, I did run upgrade migration and as a result there were some additional changes applied to fixture